boa_engine/src/builtins/object/mod.rs

```diff
@@ -57,6 +57,7 @@ impl BuiltIn for Object {
         .method(Self::has_own_property, "hasOwnProperty", 1)
         .method(Self::property_is_enumerable, "propertyIsEnumerable", 1)
         .method(Self::to_string, "toString", 0)
+        .method(Self::to_string, "toLocaleString", 0)
```

Comment:
Shouldn't this be `Self::to_locale_string`?

Reply:
Yes, copy & paste mistake :sweat_smile: